extra-python/tikzplotlib/autobuild/build

```diff
@@ -0,0 +1,4 @@
+abinfo "Installing tikzplotlib ..."
+export PIP_CONFIG_FILE=/dev/null
+pip3 install --isolated --root="${PKGDIR}" --ignore-installed --no-deps --compile .
```

Comment:
Could this not be built with the `python` ABTYPE?

Reply:
This package does not contain a `setup.py`, so the python ABTYPE is not applicable here.